views/interrupt-card.html

```diff
@@ -27,14 +27,11 @@ <h2 class="govuk-heading-m">Trusts</h2>
           <li>enable macros when prompted</li>
         </ul>
         <p class="govuk-body">  Then follow the instructions within the document to enter the information into the service. You'll need to complete a new document for each trust. </p>
-        
-        {{ govukButton({
-          text: "Continue",
-          href: "/register-an-overseas-entity/presenter",
-          attributes: {
-            "id": "continue"
-          }
-        }) }}
+
+        <a href="/register-an-overseas-entity/presenter"  id="continue" onclick="_paq.push(['trackGoal', '{{PIWIK_START_GOAL_ID}}'])" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
+          Continue
+        </a>
+
```

Comment:
for the button, have you tried adding the extra bits as attributes
{{ govukButton({
          text: "Continue",
          href: "/register-an-overseas-entity/presenter",
          attributes: {
            "id": "continue",
            "onclick": "_paq.push(['trackGoal', '{{PIWIK_START_GOAL_ID}}'])"
          }
        }) }}

maybe add data-module if you need it

Reply:
Yes I did try that but it wasn't accepting {{PIWIK_START_GOAL_ID}} as a value so had to resort to using the html version